src/firebase.js

```diff
@@ -0,0 +1,16 @@
+import { initializeApp } from 'firebase/app';
+import { getFirestore } from 'firebase/firestore';
+
+const firebaseConfig = {
+  apiKey: process.env.API_KEY,
+  authDomain: 'tcl-37-smart-shopping-list.firebaseapp.com',
+  projectId: 'tcl-37-smart-shopping-list',
+  storageBucket: 'tcl-37-smart-shopping-list.appspot.com',
+  messagingSenderId: '867331867909',
+  appId: '1:867331867909:web:c5268888f54fa923ec236c',
+};
```

Comment:
👏🏽 Well done in using environment variables!

Reply:
Love this too! We'll likely have to configure GitHub to have a [copy of this api key so our previews](https://github.com/the-collab-lab/tcl-37-smart-shopping-list/pull/17#issuecomment-1009238597) still work